src/app/zkapp_test_transaction/zkapp_test_transaction.ml

```diff
@@ -209,9 +204,9 @@ let update_state =
     Command.async
       ~summary:"Generate a zkApp transaction that updates zkApp state"
       (let%map keyfile, fee, nonce, memo, debug = Flags.common_flags
-       and snapp_keyfile =
+       and zkapp_keyfile =
          Param.flag "--zkapp-account-key"
-           ~doc:"KEYFILE Private key file to create a new zkApp account"
+           ~doc:"KEYFILE Private key file for account to get the updated state"
```

Comment:
`of the zkapp account to be updated`? Specifying which account's private key is supposed to be passed here should suffice? It is really needed for signing the zkapp party.
I think we can use the same description for all updates except create-account/deploy and upgrade commands?